packages/block-editor/src/components/alignment-control/stories/aliginment-toolbar.story.js

```diff
@@ -16,13 +16,13 @@ const meta = {
 	component: AlignmentToolbar,
 	argTypes: {
 		value: {
-			control: { type: false },
+			control: { type: undefined },
```

Comment:
I think we misread the [docs](https://storybook.js.org/docs/api/arg-types#control) 😂 Does this pass?

```suggestion
			control: false,
```

Reply:
Lol, looks like it does. Updated in https://github.com/WordPress/gutenberg/pull/67646/commits/e2b13c08106c8fd71fdebc278585460222e2e301